src/main/java/mate/academy/Main.java

```diff
@@ -27,7 +37,8 @@ public static void main(String[] args) {
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
```

Comment:
Similar to the previous comment, avoid explicit type casting by using a generic method in the `Injector`.